backend-rust/src/graphql_api/baker_metrics.sql

```diff
@@ -0,0 +1,30 @@
+SELECT
+    bucket_time.bucket_start as "bucket_time!",
+    COALESCE(after_bucket.total_bakers_added, 0) - COALESCE(before_bucket.total_bakers_added, 0) as "bucket_bakers_added!",
+    COALESCE(after_bucket.total_bakers_removed, 0) - COALESCE(before_bucket.total_bakers_removed, 0) as "bucket_bakers_removed!"
+FROM
+    date_bin_series(
+        $3::interval,
+        $2,
+        $1
+    ) AS bucket_time
+LEFT JOIN LATERAL (
+    SELECT
+        total_bakers_added,
```

Comment:
This could be MIN and MAX and also where on slot_time start of period. Since this is on query time we should do that.